File: Source/Extensions/Tessa.Extensions.Default.Shared/Workflow/KrProcess/ProcessManagementStageTypeMode.cs
namespace Tessa.Extensions.Default.Shared.Workflow.KrProcess
{
    /// <summary>
    /// Режим работы этапа "Управление процессом" (<see cref="StageTypeDescriptors.ProcessManagementDescriptor"/>).
    /// </summary>
    public enum ProcessManagementStageTypeMode
    {
        /// <summary>
        /// Переход на этап.
        /// </summary>
        StageMode = 0,

        /// <summary>
        /// Переход на группу.
        /// </summary>
        GroupMode = 1,

        /// <summary>
        /// Переход на следующую группу.
        /// </summary>
        NextGroupMode = 2,

        /// <summary>
        /// Переход на предыдущую группу.
        /// </summary>
        PrevGroupMode = 3,

        /// <summary>
        /// Переход на начало текущей группы.
        /// </summary>
        CurrentGroupMode = 4,

        /// <summary>
        /// Отправить процесс.
        /// </summary>
        SendSignalMode = 5,

        /// <summary>
        /// Отменить процесс.
        /// </summary>
        CancelProcessMode = 6,

        /// <summary>
        /// Пропустить процесс.
        /// </summary>
        SkipProcessMode = 7,
    }
}

File: WebClient SDK/src/default/ui/krProcess/processManagementStageTypeMode.ts
/**
 * Режим работы этапа "Управление процессом".
 */
export enum ProcessManagementStageTypeMode {
  /**
   * Переход на этап.
   */
  StageMode = 0,

  /**
   * Переход на группу.
   */
  GroupMode = 1,

  /**
   * Переход на следующую группу.
   */
  NextGroupMode = 2,

  /**
   * Переход на предыдущую группу.
   */
  PrevGroupMode = 3,

  /**
   * Переход на начало текущей группы.
   */
  CurrentGroupMode = 4,

  /**
   * Отправить процесс.
   */
  SendSignalMode = 5,

  /**
   * Отменить процесс.
   */
  CancelProcessMode = 6,

  /**
   * Пропустить процесс.
   */
  SkipProcessMode = 7
}
